driver/src/main/java/com/code_intelligence/jazzer/junit/JazzerFuzzTestExecutor.java

```diff
@@ -0,0 +1,181 @@
+    if (seedCorpusUrl == null) {
+      if (fuzzTest.seedCorpus().isEmpty()) {
+        // Situation: The user may not be aware of the seed corpus feature.
+        String message = String.format(
+            "Collecting crashing inputs in the project root directory.\nIf you want to keep them organized by "
+                + "fuzz test and automatically run them as regression tests with JUnit Jupiter, create a "
+                + "test resource directory called '%s' in package '%s' and move the files there.",
+            seedCorpusResourcePath, fuzzTestClass.getPackage().getName());
+        request.getEngineExecutionListener().reportingEntryPublished(
+            fuzzTestDescriptor, ReportEntry.from("seed corpus", message));
+      } else {
+        // Situation: The user explicitly configured a seed corpus, but it couldn't be found.
+        throw new FileNotFoundException(
+            String.format("Failed to find seed corpus at '%s' relative to '%s'",
+                fuzzTest.seedCorpus(), fuzzTestClass));
+      }
+    } else if ("file".equals(seedCorpusUrl.getProtocol())) {
+      // From the second positional argument on, files and directories are used as seeds but not
+      // modified.
+      libFuzzerArgs.add(seedCorpusUrl.getFile());
+      // We try to find the source tree representation of the seed corpus directory and emit
+      // findings into it.
+      findSeedCorpusDirectoryInSourceTree().ifPresent(
+          (path) -> libFuzzerArgs.add(String.format("-artifact_prefix=%s/", path)));
+    } else {
+      // We can't directly use the seed corpus from resources as it's packaged into a JAR. Instead,
+      // try to get the path to the seed corpus in the source tree.
+      Optional<Path> seedCorpusSourceDirectory = findSeedCorpusDirectoryInSourceTree();
+      if (seedCorpusSourceDirectory.isPresent()) {
+        libFuzzerArgs.add(seedCorpusSourceDirectory.get().toString());
+        // We try to find the source tree representation of the seed corpus directory and emit
+        // findings into it.
+        libFuzzerArgs.add(String.format("-artifact_prefix=%s/", seedCorpusSourceDirectory.get()));
+      } else {
+        request.getEngineExecutionListener().reportingEntryPublished(fuzzTestDescriptor,
+            ReportEntry.from("seed corpus",
+                "When running Jazzer fuzz tests from a JAR rather than class files, the seed corpus isn't used unless it is located under src/test/resources/..."));
+      }
+    }
+
+    libFuzzerArgs.add("-max_total_time=" + durationStringToSeconds(fuzzTest.maxTime()));
+    // Disable libFuzzer's out of memory detection: It is only useful for native library fuzzing,
+    // which we don't support without our native driver, and leads to false positives where it picks
+    // up IntelliJ's memory usage.
+    libFuzzerArgs.add("-rss_limit_mb=0");
+    if (request.getConfigurationParameters().getBoolean("jazzer.valueprofile").orElse(false)) {
```

Comment:
This is set via env variable? Add a test? :smile:

Reply:
It is set via the JUnit properties. Either the `junit-platform.properties` file or dedicated build system test plugin configurations (which I have never been able to figure out, but my Maven-fu is weak).

Reply:
Added an extensive test and along the way fixed an issue with IntelliJ (see last commit).